seatunnel-connectors-v2/connector-elasticsearch/src/java/org/apache/seatunnel/connectors/seatunnel/elasticsearch/client/EsRestClient.java

```diff
@@ -68,19 +70,19 @@ private static RestClientBuilder getRestClientBuilder(List<String> hosts, String
     }
 
     public static EsRestClient getInstance(List<String> hosts, String username, String password) {
-        if (restClient == null) {
+        if (REST_CLIENT == null) {
```

Comment:
I think we need `double checked locking` here.

Reply:
@iture123 Please take a look